_posts/2015-06-10-d8-theming-basics.markdown

```diff
@@ -0,0 +1,376 @@
+---
+layout: post
+title: "Drupal 8 Theming Basics"
+date: 2015-06-10
+author: Anne Tomasevich
+tags: drupal8 theming drupal
+summary: Theming in Drupal 8 means a lot of changes for current Drupalers and a lot of awesome stuff for everyone. I'll cover what's new and how to create a custom theme in Drupal 8.
+---
+
+Theming in Drupal 8 means a lot of changes for current Drupalers and a lot of awesome stuff for everyone. In this post I'll cover:
+
+- What's changing
+- The positives and negatives of these changes
+- How to create a custom theme in D8
+- Twig basics
+- Twig debugging
+
+### What's new?
+
+Too much to list here, but here are some highlights:
+
+- Twig, a template engine by SensioLabs, is used inside template files in lieu of PHP
+- Responsive design elements are included by default
+- Breakpoints can be set and used across modules and themes
+- Support for IE8 and below is dropped meaning the use of jQuery 2.0, HTML5, and CSS3 (including pseudo selectors) are now supported
+- Classy, a new base theme, is introduced
+- CSS: far fewer IDs are used, default classes are no longer in core but are moved to Classy, CSS file structure now uses SMACSS and class names follow the BEM format
+- CSS and JS files are attached to pages differently
+- template.php becomes the slightly better-named [theme-name].theme
```

Comment:
I'm not convinced it's an improvement (adding `.php` would be), but maybe I'll leave a comment about that :-)